src/participant_network/el/besu/besu_launcher.star

```diff
@@ -0,0 +1,139 @@
+load("github.com/kurtosis-tech/eth2-module/src/shared_utils/shared_utils.star", "new_port_spec", "path_join", "TCP_PROTOCOL", "UDP_PROTOCOL")
+load("github.com/kurtosis-tech/eth2-module/src/module_io/parse_input.star", "get_client_log_level_or_default")
+load("github.com/kurtosis-tech/eth2-module/src/participant_network/el/el_client_context.star", "new_el_client_context")
+
+module_io = import_types("github.com/kurtosis-tech/eth2-module/types.proto")
+
+# The dirpath of the execution data directory on the client container
+EXECUTION_DATA_DIRPATH_ON_CLIENT_CONTAINER = "/opt/besu/execution-data"
+
+GENESIS_DATA_DIRPATH_ON_CLIENT_CONTAINER = "/opt/besu/genesis"
+
+RPC_PORT_NUM = 8545
+WS_PORT_NUM = 8546
+DISCOVERY_PORT_NUM = 30303
+ENGINE_HTTP_RPC_PORT_NUM = 8550
+ENGINE_WS_RPC_PORT_NUM = 8551
+
+# Port IDs
+RPC_PORT_ID = "rpc"
+WS_PORT_ID = "ws"
+TCP_DISCOVERY_PORT_ID = "tcp-discovery"
+UDP_DISCOVERY_PORT_ID = "udp-discovery"
+ENGINE_HTTP_RPC_PORT_ID = "engineHttpRpc"
+ENGINE_WS_RPC_PORT_ID = "engineWsRpc"
+
+PRIVATE_IP_ADDRESS_PLACEHOLDER = "KURTOSIS_IP_ADDR_PLACEHOLDER"
+
+USED_PORTS = {
+	RPC_PORT_ID: new_port_spec(RPC_PORT_NUM, TCP_PROTOCOL),
+	WS_PORT_ID: new_port_spec(WS_PORT_NUM, TCP_PROTOCOL),
+	TCP_DISCOVERY_PORT_ID: new_port_spec(DISCOVERY_PORT_NUM, TCP_PROTOCOL),
+	UDP_DISCOVERY_PORT_ID: new_port_spec(DISCOVERY_PORT_NUM, UDP_PROTOCOL),
+	ENGINE_HTTP_RPC_PORT_ID: new_port_spec(ENGINE_HTTP_RPC_PORT_NUM, TCP_PROTOCOL),
+	ENGINE_WS_RPC_PORT_ID: new_port_spec(ENGINE_WS_RPC_PORT_NUM, TCP_PROTOCOL)
+}
+
+ENTRYPOINT_ARGS = ["sh", "-c"]
```

Comment:
I guess we could do the same in all the other laucnhers

Reply:
i think in some of them the containers own entrypoint isn't enough, we override it and then launch via cmd. for others the entrypoint is enough. I'll make sure that I use a constant for the entrypoint where ever its overriden in the original module.